app/views/books/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1 class="page-title"><%= t('.header') %></h1>
+
+
+<%= link_to t('.add_new_book_link'), new_book_path %>
+
+<ul class="books">
+  <% @books.each do |book| %>
+    <li> <%= t('labels.title') %> <%= book.title %> ,
```

Comment:
Usually you want to keep tags on one line or if you need to split them up, it's good to line them up. Like this:

```html
<li>
  <%= t('labels.title') %>
  <%= book.title %>
  <%= t('labels.author') %>
  <%= book.author %>
</li>
```